latest/index.bs

```diff
@@ -377,63 +377,37 @@ in which each unique pixel value represents a separate segmented object.
 `image-label` groups MUST also contain `multiscales` metadata and the two
 "datasets" series MUST have the same number of entries.
 
-The `colors` key defines a list of JSON objects describing the unique label
-values. Each entry in the list MUST contain the key "label-value" with the
-pixel value for that label. Additionally, the "rgba" key MAY be present, the
-value for which is an RGBA unsigned-int 4-tuple: `[uint8, uint8, uint8, uint8]`
-All `label-value`s must be unique. Clients who choose to not throw an error
-should ignore all except the _last_ entry.
-
-Some implementations may represent overlapping labels by using a specially assigned
+The `image-label` dictionary SHOULD contain a `colors` key whose value MUST be a
+list of JSON objects describing the unique label values. Each color object MUST
+contain the `label-value` key whose value MUST be an integer specifying the
+pixel value for that label. It MAY contain an `rgba` key whose value MUST be an tuple
```

Comment:
`a tuple` (not 'an') - Same as for 0.4/index.bs. Actually is this really a 'tuple'? In JSON you can only have a list?
"tuple" is also used in the schemas rgba descriptions

Reply:
Yes, that's definitely a Python-ism. Proposing to use `array` everywhere which is the  JSON type and also matches the underlying schema?